src/GitHubExtension/DataModel/DataObjects/PullRequest.cs

```diff
@@ -317,6 +317,26 @@ public CommitState CommitState
         }
     }
 
+    /// <summary>
+    /// Gets all reviews associated with this pull request.
+    /// </summary>
+    [Write(false)]
+    [Computed]
+    public IEnumerable<Review> Reviews
+    {
+        get
+        {
+            if (DataStore == null)

```

Comment:
This check should probably live inside GetAllForPullRequest()

Reply:
Agree that would clean up the data objects, but all of the object methods of this nature do not have DataStore as a nullable type and this pattern is used in many other objects. I would prefer to make such changes in a refactoring PR and keep this change consistent with current patterns.

Reply:
Test comment